modules/storage-persistent-storage-pv.adoc

```diff
@@ -74,7 +74,20 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 - VMware vSphere
 // - Local
 endif::openshift-enterprise,openshift-webscale,openshift-origin[]
-endif::microshift[]
+endif::microshift,openshift-rosa[]
+
+ifdef::openshift-rosa[]
+[id="types-of-persistent-volumes_{context}"]
+== Types of PVs
+
+{product-title} (ROSA) officially supports the following persistent volume storage options:
+
+- AWS Elastic Block Store (EBS)
+- AWS Elastic File Store (EFS)
+
+ROSA also functions with Kubernetes Container Storage Interface (CSI) compatible volume provisioners from third-party storage vendors such as AWS FSX or Pure Storage Portworx. Red Hat does not oversee third-party provisioners or the connected CSI drivers and the vendors fully control source code, deployment, operation, and Kubernetes compatibility. These volume provisioners are considered customer-managed and the respective vendors are responsible for providing support. See the link:https://docs.openshift.com/rosa/rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix.html#rosa-policy-responsibilities_rosa-policy-responsibility-matrix[Shared responsibilities for Red Hat OpenShift Service on AWS] matrix for more information. 
```

Comment:
From my understanding, we'd better move this description to https://70909--ocpdocs-pr.netlify.app/openshift-rosa/latest/storage/container_storage_interface/persistent-storage-csi#csi-drivers-supported_persistent-storage-csi if we want to keep consistent with the enterprise docs(keep using `Types of PVs`). The `Types of PVs` in enterprise docs specified the intree manual PVs not the CSI.

Reply:
@arendej what do you think about moving the description to the CSI page and linking over to it?

Reply:
Yeah, that will be better and clearly to customers, thanks!